### ISSUE TITLE
Custom Directives: some attributes are broken (as well as attrs with dashes) [Accessibility]

### ISSUE THREAD

**Issue by mpicard** [vuejs/Discussion#206]
v0.12.1:

As you can see in the fiddle provided, some attributes such as class, role, etc do not work properly. As well as attributes with dashes like aria-valuemin, aria-valuemax, since it is an invalid left assignment, using a string (eg. this.el.'aria-valuemin') also does not work. This obviously is a problem for accessibility.

http://jsfiddle.net/4c77uxyb/

**Comment by yyx990803** [comment]
It's a raw element, you need to use `this.el.setAttribute()`...

**Comment by mpicard** [comment]
Ah thank you so much, maybe add that to the docs, it wasn't obvious to me?
